### ISSUE TITLE
[FEATURE] Add Portfolio/Show case

### ISSUE THREAD

**Issue by wey-gu** [HEIGE-PCloud/DoIt#74]
### Describe the feature you want 描述你的功能需求

- Add page of Projects or Portfolio
Like a page or a section on the home page showing some of the projects(could be a post) :-)

Thank you PCloud for taking over loveit!

*I really liked what's offered by loveit/doit and its design language, I am planning to mitigate the "projects" with some markdown tables in a post named `projects.md` for now, just would like to check with you on whether it could be supported ;-)*

### Useful reference 有价值的参考
Something like https://rjordaney.is/lectures/ with the https://github.com/PippoRJ/hugo-refresh ?

If available, provide useful links to fulfill the feature.
如果可以的话, 提供实现这个功能的相关参考链接.

N/A

**Comment by HEIGE-PCloud** [comment]
Definitely! I think this can be supported by a new shortcode which shares the similar idea with the [`friend` shortcode](https://hugodoit.pages.dev/theme-documentation-extended-shortcodes/#12-friend) but with a different design. I will look into it and hopefully this can be supported very soon 😊

**Comment by HEIGE-PCloud** [comment]
A preview of what the `showcase` shortcode looks like.

![image](https://user-images.githubusercontent.com/52968553/119727238-f5d2ab80-be69-11eb-9fc2-321e0ae12e1e.png)

I have noticed that there is a lot of blank space on both sides. After #75 is fixed, a three-column configuration will be released.

**Comment by wey-gu** [comment]
> A preview of what the `showcase` shortcode looks like.
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/52968553/119727238-f5d2ab80-be69-11eb-9fc2-321e0ae12e1e.png)
> 
> 
> 
> I have noticed that there is a lot of blank space on both sides. After #75 is fixed, a three-column configuration will be released.
> 
> 

Dear Pcloud,
I dont know what to say, this is next level dope && making-me-cry!

You are amazing!!!

**Comment by HEIGE-PCloud** [comment]
A preview of what three-column wide configuration may look like.
![image](https://user-images.githubusercontent.com/52968553/119818604-52c07700-bee7-11eb-992b-11871f7d659c.png)

**Comment by HEIGE-PCloud** [comment]
The `showcase` shortcode has been merged into the main branch. Check it out!

[Test page](https://hugodoit.pages.dev/showcase-tests/)
[Documentation](https://hugodoit.pages.dev/theme-documentation-extended-shortcodes/#13-showcase)

**Comment by wey-gu** [comment]
Dear @HEIGE-PCloud ,

不知道说什么了，你真的太给力了，谢谢你 :-)，我放了几个内容上去回来发现你又支持column了，👍🏻。

**Comment by wey-gu** [comment]
and, it worked like a charm :)

https://siwei.io
